src/constants/constants.js

```diff
@@ -8,4 +8,7 @@ export const PASSWORD_REGEX = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
 export const AUTH_USER_KEY = 'user';
 
 // SIGN-UP GENDER
-export const GENDER_OPTIONS = ['female', 'male', 'other'];
+export const GENDER_OPTIONS = [
+  { id: 'female', label: 'FEMALE' },
```

Comment:
Do you really need the `label`in uppercase?

Reply:
my mistake, done!